SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Reporting security issues privately
+
+To report a security issue privately, please [create a security advisory](https://github.com/FSHSchool/site/security/advisories) in this repository. This will allow repository administrators to review and address it privately before public disclosure. For more details about this process, see ["Privately reporting a security vulnerability"](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability).
+
+# Project security practices
+
+The FSH School website uses NPM for dependency management. Dependencies with security vulnerabilities as reported by NPM's audit tool should be updated to secure versions as soon as possible. A new version of the FSH School website that resolves the vulnerabilities should be released as soon as possible afterwards. Pull requests that include new dependencies should not include dependencies that contain known security vulnerabilities.
```

Comment:
We typically "deploy" the site rather than "release" it.
```suggestion
The FSH School website uses NPM for dependency management. Dependencies with security vulnerabilities as reported by NPM's audit tool should be updated to secure versions as soon as possible. A new version of the FSH School website that resolves the vulnerabilities should be deployed as soon as possible afterwards. Pull requests that include new dependencies should not include dependencies that contain known security vulnerabilities.
```